packages/core/src/app/customer/CheckoutButtonContainer.tsx

```diff
@@ -46,6 +46,10 @@ const sortMethodIds = (methodIds:string[]): string[] => {
     return methodIds.sort((a, b) => order.indexOf(b) - order.indexOf(a));
 }
 
+const isPayPalCommerce = (methodId: string): boolean => {
+    return paypalCommerceIds.includes(methodId);
+}
+
```

Comment:
const isPayPalCommerce = (methodId: string): boolean => paypalCommerceIds.includes(methodId);

Reply:
Oopss, sorry for that